core/src/test/java/software/amazon/awssdk/http/SdkHttpFullResponseAdapterTest.java

```diff
@@ -23,28 +23,26 @@
 import java.io.UncheckedIOException;
 import java.io.UnsupportedEncodingException;
 import java.lang.reflect.Field;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
 import java.util.zip.GZIPInputStream;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.runners.MockitoJUnitRunner;
 import org.unitils.util.ReflectionUtils;
 import software.amazon.awssdk.util.Crc32ChecksumValidatingInputStream;
 import software.amazon.awssdk.util.StringInputStream;
+import utils.ValidSdkObjects;
 
 @RunWith(MockitoJUnitRunner.class)
 public class SdkHttpFullResponseAdapterTest {
 
-    private final SdkHttpFullRequest request = SdkHttpFullRequest.builder().build();
+    private final SdkHttpFullRequest request = ValidSdkObjects.sdkHttpFullRequest().build();
 
     @Test
     public void adapt_SingleHeaderValue_AdaptedCorrectly() throws Exception {
-        SimpleSdkHttpFullResponse httpResponse = SimpleSdkHttpFullResponse.builder()
-                                                                          .header("FooHeader", "headerValue")
-                                                                          .build();
+        SdkHttpFullResponse httpResponse = SdkHttpFullResponse.builder()
+                                                              .header("FooHeader", "headerValue")
+                                                              .statusCode(200)
```

Comment:
Why is this required now?

Reply:
To avoid `SdkHttpResponse`'s `statusCode()` from returning `Optional<Integer>`, same as the endpoint above for request. The HTTP spec requires the service to always return the status code, so making it optional felt incorrect.